### PR TITLE
CAM-11733 - Add overview of camunda:connector extension

### PR DESCRIPTION
**Pull request by SaumyaJain15**
[](https://app.camunda.com/jira/browse/)

An overview of camunda:connector extension on this page will help users to understand how to use connectors for calling APIs in workflow.

### CODE REVIEW

**Comment by marstamm** [comment]
Hi @SaumyaJain15 ,

thank you for your contribution. We will review your additions shortly and come back to you.

I have created the ticket CAM-11733 where you can track the progress.

Cheers,
Martin

**Comment by mboskamp** [comment]
Hey @SaumyaJain15,

I haven't heard from you back for a while now. Please let me know by the end of this week if you want to proceed with this. Otherwise I will close the PR.

Cheers and have a nice week.
Miklas

**Comment by SaumyaJain15** [comment]
Hi @mboskamp,  

Sorry for the delayed response. I have made changes according to your review comments. Please check if this looks fine. I'll raise a PR to master branch once this is approved.

Thanks,
Saumya Jain

File: content/reference/bpmn20/tasks/service-task.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ For more information about expression language as delegation code, please see th
 [section]({{< ref "/user-guide/process-engine/expression-language.md#use-expression-language-as-delegation-code" >}})
 of the [User Guide]({{< ref "/user-guide/_index.md" >}}).
 
+It is also possible to invoke logic which is implemented in form of webservices. camunda:connector is an extension that allows calling REST/SOAP APIs directly from the workflow. For more information about using connectors, please see the corresponding [section]({{< ref "/user-guide/process-engine/connectors.md#use-connectors" >}}) of the [User Guide]({{< ref "/user-guide/_index.md" >}}).
```
Comment (mboskamp) [inline content/reference/bpmn20/tasks/service-task.md:60]:
```suggestion
It is also possible to invoke logic which is implemented in form of webservices. `camunda:connector` is an extension that allows calling REST/SOAP APIs directly from the workflow. For more information about using connectors, please see the corresponding [section]({{< ref "/user-guide/process-engine/connectors.md#use-connectors" >}}) of the [User Guide]({{< ref "/user-guide/_index.md" >}}).
```

**Comment by SaumyaJain15** [comment]
Hi @mboskamp

Have created a PR to merge the approved changes to master. https://github.com/camunda/camunda-docs-manual/pull/533

Thanks,
Saumya

**Comment by mboskamp** [comment]
Thanks @SaumyaJain15, 
I won't get to it before next week. I'll keep you posted.

Cheers,
Miklas

**Comment by mboskamp** [comment]
Merged with 406e201f305a9bf9413b774969b0e9dd1934308c

**Comment by SaumyaJain15** [comment]
Thanks @mboskamp

**Pull request closed without merging**